styles/pages/TorchboxAcademy.module.scss

```diff
@@ -0,0 +1,49 @@
+.container {
+    max-width: 1440px;
+    margin: 0 auto;
+}
+
+.textContainer {
```

Comment:
**Suggestion**: I would delete this if it's not being used

Reply:
This is being used on another branch, I saved this file locally so I have the code from a previous MR